packages/perseus-editor/src/widgets/__stories__/interactive-graph-editor.stories.tsx

```diff
@@ -1,123 +1,110 @@
+import {action} from "@storybook/addon-actions";
 import * as React from "react";
 
-import {flags} from "../../__stories__/flags-for-api-options";
+import {apiOptionsWithDefaults} from "../../__stories__/flags-for-api-options";
 import InteractiveGraphEditor from "../interactive-graph-editor/interactive-graph-editor";
 import {getDefaultFigureForType} from "../interactive-graph-editor/locked-figures/util";
 
-import InteractiveGraphEditorArgTypes from "./interactive-graph-editor.argtypes";
-
 import type {Meta, StoryObj} from "@storybook/react";
 
 const mafsOptions = {
-    apiOptions: {
-        flags,
-    },
+    apiOptions: apiOptionsWithDefaults,
     graph: {
-        type: "segment",
+        type: "segment" as const,
     },
     correct: {
-        type: "segment",
+        type: "segment" as const,
     },
 };
 
 const defaultPointProps = getDefaultFigureForType("point");
 
-export default {
+const meta: Meta = {
     title: "PerseusEditor/Widgets/Interactive Graph Editor",
     component: InteractiveGraphEditor,
-    argTypes: InteractiveGraphEditorArgTypes,
-} as Meta<typeof InteractiveGraphEditor>;
-
-export const Default = (args): React.ReactElement => {
-    return <InteractiveGraphEditor {...args} />;
+    args: {
+        box: [3000, 3000],
+        gridStep: [1, 1],
+        labels: ["x", "y"],
+        markings: "graph",
+        range: [
+            [-10, 10],
+            [-10, 10],
+        ],
+        rulerLabel: "",
+        rulerTicks: 10,
+        showProtractor: false,
+        showRuler: false,
+        showTooltips: false,
+        snapStep: [1, 1],
+        step: [1, 1],
+        onChange: action("onChange"),
+    },
+    argTypes: {
+        // Hide controls that don't make sense to edit, or aren't easily
+        // editable, in Storybook.
+        apiOptions: {table: {disable: true}},
+        backgroundImage: {table: {disable: true}},
+        correct: {table: {disable: true}},
+        graph: {table: {disable: true}},
+        onChange: {table: {disable: true}},
+        valid: {table: {disable: true}},
+        // Readonly controls - so we can see what the args are
+        lockedFigures: {table: {readonly: true}},
+    },
 };
+export default meta;
 
-type StoryComponentType = StoryObj<typeof InteractiveGraphEditor>;
+type Story = StoryObj<typeof InteractiveGraphEditor>;
 
-// Set the default values in the control panel.
-Default.args = {
-    box: [288, 288],
-    gridStep: [1, 1],
-    labels: ["x", "y"],
-    markings: "graph",
-    range: [
-        [-10, 10],
-        [-10, 10],
-    ],
-    rulerLabel: "",
-    rulerTicks: 10,
-    showProtractor: false,
-    showRuler: false,
-    showTooltips: false,
-    snapStep: [1, 1],
-    step: [1, 1],
-};
+function StatefulDecorator(Story, context) {
```

Comment:
This StatefulDecorator does what each of the stories did in `render`... in one place. :)